docs/howtos/cluster-resource-override.md

```diff
@@ -0,0 +1,208 @@
+## API Components
+The ResourceOverride API consists of the following components:
+- **Cluster Resource Selectors**: These specify the set of cluster resources selected for overriding.
+- **Policy**: This specifies the policy to be applied to the selected resources.
+
+
+The following sections discuss these components in depth.
+
+## Cluster Resource Selectors
+A `ClusterResourceOverride` object may feature one or more cluster resource selectors, specifying which resources to select to be overridden.
+
+The `ClusterResourceSelector` object supports the following fields:
+- `group`: The API group of the resource
+- `version`: The API version of the resource
+- `kind`: The kind of the resource
+- `name`: The name of the resource
+> Note: The resource can only be selected by name.
+
+
+To add a resource selector, edit the `clusterResourceSelectors` field in the `ClusterResourceOverride` spec:
+```yaml
+apiVersion: placement.kubernetes-fleet.io/v1alpha1
+kind: ClusterResourceOverride
+metadata:
+  name: example-cro
+spec:
+  clusterResourceSelectors:
+    - group: rbac.authorization.k8s.io
+      kind: ClusterRole
+      version: v1
+      name: secret-reader
+```
+
+The example above will pick a `ClusterRole` named `secret-reader` to be overridden.
+
+
+## Policy
+The `Policy` is made up of a set of rules (`OverrideRules`) that specify the changes to be applied to the selected
+resources on selected clusters.
+
+Each `OverrideRule` supports the following fields:
+- **Cluster Selector**: This specifies the set of clusters to which the override applies.
+- **JSON Patch Override**: This specifies the changes to be applied to the selected resources.
+
+To add an override rule, edit the `policy` field in the `ClusterResourceOverride` spec:
+
+```yaml
+apiVersion: placement.kubernetes-fleet.io/v1alpha1
+kind: ClusterResourceOverride
+metadata:
+  name: example-cro
+spec:
+  clusterResourceSelectors:
+    - group: rbac.authorization.k8s.io
+      kind: ClusterRole
+      version: v1
+      name: secret-reader
+  policy:
+    overrideRules:
+      - clusterSelector:
+          clusterSelectorTerms:
+            - labelSelector:
+                matchLabels:
+                  env: prod
+        jsonPatchOverrides:
+          - op: remove
+            path: /rules/0/verbs/2
+```
+The `ClusterResourceOverride` object above will remove the verb "list" in the `ClusterRole` named `secret-reader` on 
+clusters with the label `env: prod`.
+
+### Cluster Selector
+To specify the clusters to which the override applies, you can use the `clusterSelector` field in the `OverrideRule` spec.
+The `clusterSelector` field supports the following fields:
+- `clusterSelectorTerms`: A list of terms that are used to select clusters.
+    * Each term in the list is used to select clusters based on the label selector.
+
+### JSON Patch Override
+To specify the changes to be applied to the selected resources, you can use the `jsonPatchOverrides` field in the `OverrideRule` spec.

```

Comment:
Can we mention that "JSONPatchOverride applies a JSON patch on the selected resources following [RFC 6902](https://datatracker.ietf.org/doc/html/rfc6902)."?

All the fields defined follow this RFC.